packages/desktop-client/src/components/spreadsheet/useFormat.ts

```
@@ -30,18 +33,29 @@ function format(
+      }
+      return formatter.format(value);
     case 'percentage':
       return value + '%';
     case 'financial-with-sign':
-      const formatted = format(value, 'financial', formatter);
+      const formatted = format(
+        value,
+        'financial',
+        formatter,
+        effectiveNaturalDecimalPlaces,
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)